plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3ClientSettings.java

```diff
@@ -205,7 +205,7 @@ final class S3ClientSettings {
     static final Setting.AffixSetting<TimeValue> CONNECTION_ACQUISITION_TIMEOUT = Setting.affixKeySetting(
         PREFIX,
         "connection_acquisition_timeout",
-        key -> Setting.timeSetting(key, TimeValue.timeValueMinutes(2), Property.NodeScope)
+        key -> Setting.timeSetting(key, TimeValue.timeValueMinutes(20), Property.NodeScope)
```

Comment:
Maybe add some comments on why 20 minutes is a good timeout?  Also, maybe make it a constant.

Reply:
It shouldn't be a constant. This is a repository setting and users should be able to override it. There's no one specific good configuration. This is just to handle upload bursts in special cases like force merge temporarily.